server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/AbstractGroupUpdate.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
I would not mind having parameters repeated in both classes, even if they happen to have the same parameters